clinica/iotools/converters/adni_to_bids/adni_utils.py

```diff
@@ -715,6 +715,8 @@ def create_adni_sessions_dict(
                 ]
                 df_subj_session = pd.concat([df_subj_session, df_filtered], axis=1)
 
+    if df_subj_session.empty:
+        raise Exception("Empty dataset detected. Clinical data can not be extracted")
```

Comment:
I'd suggest to use a `ValueError` instead of the generic `Exception` type. The latter is meant to be subclassed rather than used directly.

Reply:
```suggestion
        raise ValueError("Empty dataset detected. Clinical data can not be extracted")
```

Reply:
Good call. Changed.